packages/synthetics-sdk-broken-links/src/link_utils.ts

```diff
@@ -182,6 +182,7 @@ function parseFollowedLinks(
   };
 
   for (const link of followed_links) {
+    if (link.link_passed === undefined) continue;
```

Comment:
Is this to handle us not ever visiting a link due to timeout?

Reply:
This is to handle if the origin link is not done being checked due to timeout. 

If a regular link is stopped because of timeout it is not added to list of checked_links